rummager/config/deploy.rb

```diff
@@ -54,11 +54,17 @@
   task :restart_published_content_listener do
     run "sudo initctl restart rummager-govuk-index-queue-listener-procfile-worker || sudo initctl start rummager-govuk-index-queue-listener-procfile-worker"
   end
+
+  desc "Restart rummager's publishing-api bulk reindex listener for govuk-index"
+  task :restart_published_content_bulk_reindex_listener do
```

Comment:
If we add any more listener tasks we should ask @JonathanHallam if he wants to metaprogram them. 🙃